bosh-releases/cf-operator-testing-release/jobs/failing-drain-job/templates/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+trap 'ls -l /tmp/drain_logs; exit 0' TERM
```

Comment:
should be exit 1?

Reply:
It's the drain script that should exit non-zero.